CMakeLists.txt

```diff
@@ -84,6 +82,10 @@ endif()
 add_subdirectory(data)
 
 add_subdirectory(game)
+
+target_compile_options(performous PRIVATE -Wall $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:-Wextra> $<$<CXX_COMPILER_ID:gcc>:-fcx-limited-range> $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:-Wconversion>)
```

Comment:
As said on Discord, we shouldn't add flags like this since they can differ between compilers. Currently noted with the `NOT:COMPILER_ID:MSVC` for example. 
If possible make use of cmake compile features list: https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html#prop_gbl:CMAKE_CXX_KNOWN_FEATURES

this will make sure the correct compile option is added for the correct compiler

Reply:
I mean sure, but I just copy-pasted from a previous `add_compile_features` which is objectively worse. 👀😂

Reply:
ok, let's move on an fix this in a separate MR